backend/conftest.py

```diff
@@ -44,6 +44,7 @@ def super_client(client, django_user_model, super_credentials):
 @pytest.fixture
 def sparqlstore(settings):
     store = settings.RDFLIB_STORE
+    store.update('CLEAR ALL')
     assert not store.query(HAS_TRIPLES)
```

Comment:
If you enforce that the store is empty, you probably don't need to assert it.

(I have no clear preference right now whether you should enforce or assert it.)

Reply:
I think the `assert` can go. This fixture causes trouble when a test is not fully completed, for example because the developer manually exits the debug session. The triplestore teardown doesn't take place, and subsequent will not start because of the `assert` (which is a good thing, a lot of tests blow up when there are leftover triples).

Reply:
A potential risk in running clear is if you haven't configured your testing environment to use a different RDFLIB_STORE. This seems unlikely, though, I think you would have to circumvent the existing test setup to achieve that.

Reply:
Also, as long as we don't run the unittests in production, no important data should be deleted. At worst, the developer needs to create new sandbox data.